exchanges/alphapoint/alphapoint.go

```
@@ -191,7 +191,7 @@ func (a *Alphapoint) CreateAccount(firstName, lastName, email, phone, password s
 
 	err := a.SendAuthenticatedHTTPRequest("POST", alphapointCreateAccount, request, &response)
 	if err != nil {
-		log.Println(err)
+		log.Error(err)
```

Might be better to return that error